sw/airborne/modules/computer_vision/opencv_example.cpp

```diff
@@ -34,26 +34,27 @@ using namespace std;
 using namespace cv;
 #include "opencv_image_functions.h"
 
+
 int opencv_example(char *img, int width, int height)
 {
   // Create a new image, using the original bebop image.
   Mat M(height, width, CV_8UC2, img);
   Mat image;
   // If you want a color image, uncomment this line
-//   cvtColor(M, image, CV_YUV2BGR_Y422);
+  cvtColor(M, image, CV_YUV2BGR_Y422);
   // For a grayscale image, use this one
-  cvtColor(M, image, CV_YUV2GRAY_Y422);
+//  cvtColor(M, image, CV_YUV2GRAY_Y422);
 
   // Blur it, because we can
-//  blur(image, image, Size(5, 5));
+  blur(image, image, Size(5, 5));
 
   // Canny edges, only works with grayscale image
-  int edgeThresh = 35;
-  Canny(image, image, edgeThresh, edgeThresh * 3);
+//  int edgeThresh = 35;
+//  Canny(image, image, edgeThresh, edgeThresh * 3);
 
   // Convert back to YUV422, and put it in place of the original image
-  grayscale_opencv_to_yuv422(image, img, width, height);
-//  colorrgb_opencv_to_yuv422(image, img, width, height);
+//  grayscale_opencv_to_yuv422(image, img, width, height);
+  colorbgr_opencv_to_yuv422(image, img, width, height);
```

Comment:
It may be nice to put the greyscale and color example in an if else structure to make things more clear.

Reply:
Agreed, fixed.